gpkit/geometric_program.py

```diff
@@ -52,13 +52,15 @@ def __init__(self, cost, constraints, substitutions=None, verbosity=1):
         ## Create list of substituted posynomials <= 1
         self.posynomials = [cost.sub(self.substitutions)]
         self.constr_idxs = []
-        self.unusedsubkeys = set(key for key in self.substitutions
-                                 if (key not in constraints.varkeys
-                                     and key not in self.cost.varkeys))
+        self.unusedsubkeys = set()
+        # self.unusedsubkeys = set(key for key in self.substitutions
+        #                          if (key not in constraints.varkeys
+        #                              and key not in self.cost.varkeys))
```

Comment:
inline comment explaining what's going on here and what should eventually happen with the commented out code?